inc/endpoints/class-form-endpoint.php

```diff
@@ -46,8 +46,9 @@ public function register_rest_routes() {
 				self::get_namespace(),
 				'/form/' . $endpoint['slug'],
 				[
-					'methods'  => \WP_REST_Server::CREATABLE,
-					'callback' => $endpoint['callback'],
+					'callback'            => $endpoint['callback'],
+					'methods'             => \WP_REST_Server::CREATABLE,
+					'permission_callback' => '__return_true',
```

Comment:
Similar to the caching endpoint, I don't think POST endpoints should ever be public, but I'm unsure what the correct capabilities should be for this endpoint.

Reply:
This endpoint is used for public POSTs to forms, ex. contact form. Being public is expected.

Reply:
Ah, thanks. My misunderstanding.